app/src/main/java/org/wikipedia/LongPressHandler.kt

```diff
@@ -20,7 +20,7 @@ import org.wikipedia.util.UriUtil.isValidPageLink
 
 class LongPressHandler(view: View, private val historySource: Int, private val callback: LongPressMenu.Callback) : OnCreateContextMenuListener, OnTouchListener {
     interface WebViewMenuCallback : LongPressMenu.Callback {
-        val wikiSite: WikiSite
```

Comment:
Why can't we keep `wikiSite` nonnull?

Reply:
The `override val wikiSite = fragment.title?.wikiSite` in `PageContainerLongPressHandler` has a chance to be a `null` if the `fragment.title` returns `null`.

We can do either put `!!` after `wikiSite` or change the callback to become a nonnull type of variable.